src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstaticimport/InputAvoidStaticImportDefault.java

```diff
@@ -7,22 +7,13 @@
 
 package com.puppycrawl.tools.checkstyle.checks.imports.avoidstaticimport;
 
-import com.puppycrawl.tools.checkstyle.checks.imports.*;
-
 import java.io.*;
 import java.lang.*;
 import java.sql.Connection;
-import java.util.List;
```

Comment:
@suniti0804 is is not clear why we need to remove java.util, java.eat, and javax (swing) imports, please explain.

Reply:
it is not part of any violation and is also an unused import statement